support-frontend/assets/pages/showcase/components/ctaContribute.jsx

```diff
@@ -42,6 +43,7 @@ export default function CtaContribute() {
               icon={<ArrowRightStraight />}
               appearance="secondary"
               href="/contribute"
+              onClick={() => sendClickedEvent('support_page_cta_contribute')()}
```

Comment:
Are you going to put a similar event on the subscribe button?

Reply:
Yes, I have done :-)

Reply:
See below :down

Reply:
🤦‍♂ Oh yeah!